### PR TITLE
Refactor Click to chat LRQA-82307 | Master

### PR DESCRIPTION
**Pull request by MGabriellaS**
[LRQA-82307](https://liferay.atlassian.net/browse/LRQA-82307)

### CODE REVIEW

**Comment by MGabriellaS** [comment]
ci:test:sf

**Comment by MGabriellaS** [comment]
ci:test:sf

File: modules/apps/click-to-chat/click-to-chat-test/src/testFunctional/tests/ClickToChatInstanceSettings.testcase [hunk 1]
```diff
@@ -252,36 +250,30 @@ definition {
...
-		Select(
-			locator1 = "ClickToChatConfiguration#CHAT_PROVIDER_ID",
-			value1 = "Hubspot");
-
-		var clickToChatHubspotPassword = PropsUtil.get("click.to.chat.hubspot.password");
-
-		Type(
-			locator1 = "ClickToChatConfiguration#CHAT_PROVIDER_ACCOUNT_ID",
-			value1 = ${clickToChatHubspotPassword});
+		ClickToChatUtils.fillGoalsChat(
```
Comment (nikki-pru) [inline modules/apps/click-to-chat/click-to-chat-test/src/testFunctional/tests/ClickToChatInstanceSettings.testcase:272]: Missing task blocks

**Review by nikki-pru — CHANGES_REQUESTED** [review]
Thanks!

Not sure if it will be addressed in a different PR but the test `HubspotProviderKeepsEnabledAfterLogout` is not yet refactored in this file.

**Comment by MGabriellaS** [comment]
ci:test:sf

File: modules/apps/click-to-chat/click-to-chat-test/src/testFunctional/tests/ClickToChatInstanceSettings.testcase (hunk 1 above)
Comment (MGabriellaS) [reply modules/apps/click-to-chat/click-to-chat-test/src/testFunctional/tests/ClickToChatInstanceSettings.testcase]: The test  that was missing tasks was: `HubspotCanBeEnabledAndDisabled` I updtaed this now.  And the test: `HubspotProviderKeepsEnabledAfterLogout` is has been updated in the other PR. Thank you Nikki.

**Comment by MGabriellaS** [comment]
ci:test:sf

**Comment by nikki-pru** [comment]
ci:forward

**Pull request closed without merging**